framework/SupervisedLearning/KerasRegression.py

```diff
@@ -145,7 +151,6 @@ def evaluate(self,edict):
           featureValuesShape = fval.shape
         if featureValuesShape != fval.shape:
           self.raiseAnError(IOError,'In training set, the number of values provided for feature '+feat+' are not consistent to other features!')
-        self._localNormalizeData(values,names,feat)
```

Comment:
This was a bug since it changed the normalization constants based on the evaluation data.